controllers/components/dashboard/dashboard_controller.go

```diff
@@ -105,16 +109,11 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 		WithAction(customizeResources).
 		WithAction(deploy.NewAction(
 			deploy.WithCache(),
-			deploy.WithFieldOwner(componentsv1.DashboardInstanceName),
-			deploy.WithLabel(labels.ComponentPartOf, componentsv1.DashboardInstanceName),
-		)).
-		WithAction(updatestatus.NewAction(
-			updatestatus.WithSelectorLabel(labels.ComponentPartOf, componentsv1.DashboardInstanceName),
```

Comment:
I have a tangential question - Is there a reason for labels to be `components.opendatahub.io/part-of:default-dashboard` instead of `components.opendatahub.io/part-of:dashboard` ?

Reply:
i guess the question is:
should the value be the name of the component CR or the name of the component kind ?

Reply:
Yeah, the way we use labels should be cleaned up _a little bit_ :) , so as today we use the instance name because that label is used by predicates and event mapper and some actions but I think the right path would be to:

- use the `Kind` for the `part-of` 
- add some additional annotations for other stuffs (predicated, mapper, etc)